src/plugins/sails.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const Tags = require('opentracing').Tags
+
+function wrapAction (action, tracer, identity) {
+  return function actionWithTracing (req, res, next) {
+    const classicAction = typeof action === 'function'
+
+    const scope = tracer.scope()
+    const childOf = scope.active()
+    const span = tracer.startSpan(
```

Comment:
Where do these spans reside w/ those of express and http?

Reply:
I think these are handled as express middleware, so express spans are before and after it